pages/review/[lesson].tsx

```diff
@@ -40,19 +41,22 @@ const Review: React.FC<QueryDataProps<AppData>> = ({ queryData }) => {
     return <LoadingSpinner />
   }
   if (!session) {
-    router.push('/login')
-    return null
+    return (
```

Comment:
When there's no session that means the user is not logged in, so I think it would be a better `user experience` for people to be redirected to the login screen instead of receiving an `Error` display.

Reply:
This same comment applies to the curriculum page when no data is shown.

Reply:
No session redirects to login page now.